dbms/src/Flash/Coprocessor/StreamingDAGResponseWriter.cpp

```diff
@@ -124,6 +158,8 @@ void StreamingDAGResponseWriter<StreamWriterPtr>::encodeThenWriteBlocks(
             {
                 chunk_codec_stream->encode(block, 0, block.rows());
                 packet.add_chunks(chunk_codec_stream->getString());
+                // Fine grained shuffle is disabled, all packet need to be handled by receiver's first stream.
+                packet.add_stream_ids(0);
```

Comment:
What does this comment means?

Reply:
Originally, I wanted to let distinguish old and new version of tiflash by checking if `MPPDataPacket.stream_ids` is setted or not.(New version tiflash always set `MPPDataPacket.stream_ids`. Old version doesn't).

Reply:
But looks like only hash partition ExchangeSender need to distinguish different tiflash version. So I think we can delete this line.